app/controllers/fields_controller.rb

```diff
@@ -57,7 +57,7 @@ def find_transformation_definition
   end
 
   def find_fields
-    @fields = @transformation_definition.fields.where(id: params['fields'])
+    @fields = @transformation_definition.fields.where(id: params['fields']).order(created_at: :desc)
```

Comment:
What is this change for? Does it need a unit test?

Reply:
This is for a change that was lost on the front end that I fixed here while I was here. I will add a test so it doesn't go missing again :)